src/lib.rs

```diff
@@ -355,6 +355,8 @@ pub use crate::{
 };
 #[cfg(feature = "csfle")]
 pub use crate::client::csfle::client_encryption;
+#[cfg(feature = "csfle")]
+pub use ::mongocrypt;
```

Comment:
I imagine this will need to be rebased to make use of the new feature flag name.

Reply:
Yup, done.